spec/controllers/jasmine_fixtures/contacts_spec.rb

```diff
@@ -11,6 +11,8 @@
       @aspect = bob.aspects.create(:name => "another aspect")
       bob.share_with alice.person, @aspect
       bob.share_with eve.person, @aspect
+      # not mutual contact
+      bob.share_with(FactoryGirl.create(:person), @aspect)
```

Comment:
It looks like adding this to the `before` has side-effects to other tests which use the another fixture generated here (see [travis](https://travis-ci.org/diaspora/diaspora/jobs/356786625)). I think it's probably the easiest to just move this line to the `generates the aspects_manage_contacts_json fixture` again where it was before.

Reply:
Hmm. Everything is green here: https://travis-ci.org/diaspora/diaspora/builds/356791736?utm_source=github_status&utm_medium=notification

Reply:
Yes, it doesn't fail always, but it fails randomly, and it's at a place where you changed something, so I think it's related (because these tests worked stable before).